examples/with-evm-wagmi/src/components/SignMessage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react'
+import { recoverMessageAddress } from 'viem'
+import { type Address, useSignMessage } from 'wagmi'
+
+export function SignMessage() {
+  const [recoveredAddress, setRecoveredAddress] = useState<Address>()
+  const {
+    data: signature,
+    variables,
+    error,
+    isLoading,
+    signMessage,
+  } = useSignMessage()
+
+  useEffect(() => {
+    ;(async () => {
+      if (variables?.message && signature) {
+        const recoveredAddress = await recoverMessageAddress({
```

Comment:
this function throw error

Reply:
fixed in [fix: remove recoverTypedDataAddress & recoverMessageAddress](https://github.com/blocto/blocto-sdk/pull/228/commits/3f15d41250be27fb9c834872d99b691015280eb2)